src/Searchbar.vue

```diff
@@ -15,38 +15,90 @@ export default {
   },
   computed: {
     primitiveData() {
+      function allMatch(info, regexes) {
+        let matched = true;
+        regexes.forEach((regex) => {
+          if (!regex.test(info)) {
+            matched = false;
+          }
+        });
+        return matched;
+      }
```

Comment:
Use [`Array.prototype.every`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every):

```js
regexes.every(regex => regex.test(info));
```

In addition, it does early return.